repairnator/docker-images/kubernetes/pipeline-dockerimage/projects_to_ignore.txt

```diff
@@ -0,0 +1 @@
+surli/failingproject
```

Comment:
We have a [blacklist](https://github.com/Spirals-Team/repairnator/tree/master/repairnator/repairnator-realtime/src/main/resources) and an option to specify a blacklist when running rt-scanner.